react/ReadMore/ReadMore.js

```diff
@@ -30,7 +30,8 @@ type Props = {|
   moreLabel: string,
   lessLabel: string,
   backgroundComponentName: 'card' | 'body' | 'gray-lightest',
-  onShowMore?: Function
+  onShowMore?: Function,
+  onMoreButtonVisibilityChange?: Function
```

Comment:
Are there other ways for visibility to change other than clicking the more button? Would `onVisibilityChange` suffice?

Reply:
I see what the hook is for now. Disregard this :)